FreeRTOS/FreeRTOS
# Enable release from forks

Description
-----------
This PR updates the hardcoded github organization in the release CI script to dynamically fetch it from the github context, enabling testing releases from forked branches.

Test Steps
-----------


Checklist:
----------


- [*] I have tested my changes. No regression in existing tests.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
